statistics-dashboard-common/src/main/java/statistics/dashboard/common/models/StatisticsFilteringRequest.java

```diff
@@ -0,0 +1,115 @@
+package statistics.dashboard.common.models;
```

Comment:
There seems to be some naming confusion in the package names here... what is a model and what is a view? Maybe I'm not understanding something.

In fact it looks like your definition for 'model' is a request object, whereas 'view' is a response? Actually, not quite... there are some exceptions. Anyway, maybe we can replace model and view with request and response?

In Metis we're using (or at least trying to use) 'model' for database models, and 'view' for classes that are exchanged with the UI.

Reply:
Ah, and something else: as mentioned before, we should start ALL packages with `eu.europeana`.

Reply:
I used the name 'view' with the same understanding. I just realized that one of the classes was put in the wrong place 
I used the name 'model' as the package that will contain objects that will (or are planned to) be used for the rest of the project, which would include database models as well, but as a sub-package.
I understand the confusion, but not sure if those name packages are the best. Perhaps this explanations gave some clarification or other name ideas?

I will add the `eu.europeana` through as packages then